ChameleonForms/ModelBinders/FlagsEnumModelBinder.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+using System.Web.Mvc;
+
+namespace ChameleonForms.ModelBinders
+{
+    /// <summary>
+    /// Binds a flags enum in a model.
+    /// </summary>
+    public class FlagsEnumModelBinder : DefaultModelBinder

```

Comment:
Flags enums don't bind properly with the default model binder (only the first value is bound).